examples/mxnet/README.md

```diff
@@ -1,4 +1,13 @@
 # Model Examples using DGL (w/ MXNet backend)
+```
+use DGLBACKEND=mxnet to convert DGLBACKEND to mxnet.
+```
```

Comment:
Code block should only have code in it. Change to:
use `DGLBACKEND=mxnet` to use MXNet as DGL's backend

Reply:
I have fixed that.

Reply:
the DGLBACKEND=mxnet should be wrapped with a pair of backquote, if you are not familiar with that, check the following:
```
Use `DGLBACKEND=mxnet` to use MXNet as DGL's backend.
```

Reply:
Done